packages/tradershub/src/AppContent.tsx

```diff
@@ -1,19 +1,22 @@
 import React, { Fragment } from 'react';
+import { useLocation } from 'react-router-dom';
 import { AppContainer, EUDisclaimerMessage } from '@/components';
 import { RealAccountCreation } from '@/flows';
 import { useRegulationFlags } from '@/hooks';
 import { Router } from '@/routes';
 
 const AppContent = () => {
     const { isEU } = useRegulationFlags();
+    const location = useLocation();
+    const compareAccountsRoute = location.pathname === '/traders-hub/compare-accounts';
 
     return (
         <Fragment>
-            <AppContainer>
-                <div className='z-10' id='v2_modal_show_header_root' />
+            <div className='z-10' id='v2_modal_show_header_root' />
+            <AppContainer className={compareAccountsRoute ? 'max-w-[800px] lg:pt-0 lg:pb-20' : ''}>
```

Comment:
I feel like there can be a better way to handle the width here. Can't we implement the width on the content in this route rather adding here. adding here seems unsafe for some reason

Reply:
I tried, but the AppContainer is the parent and i cant seem to overwrite it